scripts/oracle.ts

```diff
@@ -137,23 +143,31 @@ export async function attestationsFromEvent(event: Event): Promise<OracleData[]>
   }));
 }
 
+const sequencer = getRequiredEnv("STARKNET_SEQUENCER");
+console.log(`Starting oracle on ${sequencer}`);
 http.createServer(async (req, res) => {
```

Comment:
Maybe use sth like: https://github.com/shunanya/logTracker. No need to reinvent the wheel.